game_world.py

```diff
@@ -497,7 +502,7 @@ def city_menu(character):
             print("6. Purchase a home")
             print("7. Buy a bounty map")
             print("8. Join the caravan to Flats Lagoon")
-            print("9. Quit to the main menu")
+            print("9. Exit the game")

```

Comment:
I would use "Save and exit" to help communicate the benefit choosing this option; as opposed to killing the program or closing the window.

It's clear to us in this PR that state will be saved, but the end-user may not be aware.

Reply:
Thank you, forgot to change that